packages/ilios-common/addon/components/sessions-grid-loading.hbs

```diff
@@ -24,7 +24,7 @@
         {{random 1 99}}
       </span>
       <span class="session-grid-first-offering">
-        {{moment "2005-06-24" "L LT"}}
+        {{format-date (now)}}
```

Comment:
I can't figure out where the `now` helper is coming from. It works, I'd just like to know why...

Reply:
it's part of `ember-intl`, see 

https://www.npmjs.com/package/ember-intl/v/4.2.0#format-date

Reply:
ugh, nevermind. this was supposedly removed in v5 of `ember-intl`. we're currently on v6.

https://ember-intl.github.io/ember-intl/docs/legacy/migration-4-0-to-5-0#intl-relativetimeformat

but it's evidently still working. looking into it...

Reply:
handled in the most inelegant way possible :white_check_mark: